lib/fs/fs.c

```diff
@@ -34,9 +36,20 @@ result_t trn_fs_set_root(trn_inode_t *new_root) {
 	return RESULT_OK;
 }
 
+// This will only work if trn_fs_set_root has not been called!
+result_t trn_fs_mount(const char *mount_name, trn_inode_t *mount) {
+	if (root != NULL)
+		return trn_rootfs_mount_fs(root, mount_name, mount);
```

Comment:
if we don't remove this, can we find a way to verify that root is indeed a rootfs?

Reply:
nevermind, looks like trn_rootfs_mount_fs does this already